Bottom: 5851def43be697a39cb8a99adbbd85a430f4e353
Top:    40821102c45434437b57fef0005aa6c4758c12b5
Author: Hal Rosenstock <hal@dev.mellanox.co.il>
Date:   2013-09-11 15:37:11 -0400

[librdmacm] man/rstream.1: Update man page to be consistent with rstream -h

Signed-off-by: Hal Rosenstock <hal@mellanox.com>
Signed-off-by: Sean Hefty <sean.hefty@intel.com>


---

diff --git a/man/rstream.1 b/man/rstream.1
index 29aa66b..f53679f 100644
--- a/man/rstream.1
+++ b/man/rstream.1
@@ -4,8 +4,8 @@ rstream \- streaming over RDMA ping-pong test.
 .SH SYNOPSIS
 .sp
 .nf
-\fIrstream\fR [-s server_address] [-b bind_address] [-B buffer_size]
-			[-I iterations] [-C transfer_count]
+\fIrstream\fR [-s server_address] [-b bind_address] [-f address_format]
+			[-B buffer_size] [-I iterations] [-C transfer_count]
 			[-S transfer_size] [-p server_port] [-T test_option]
 .fi
 .SH "DESCRIPTION"
@@ -21,6 +21,9 @@ This option must be specified by the client.
 \-b bind_address
 The local network address to bind to.
 .TP
+\-f address_format
+Supported address formats are ip, ipv6, gid, or name.
+.TP
 \-B buffer_size
 Indicates the size of the send and receive network buffers.
 .TP
@@ -52,6 +55,8 @@ f | fork - fork server processing (forces -T s option)
 .P
 n | nonblocking - uses non-blocking calls
 .P
+r | resolve - use rdma cm to resolve address
+.P
 v | verify - verifies data transfers
 .SH "NOTES"
 Basic usage is to start rstream on a server system, then run
